src/ts/components/account/LoginFooter.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import t from '../../services/i18n'
+import { Footer, FooterLink } from '../basic-components'
+
+export default () => {
+  const loginFooter = {
+    width: '40%',
```

Comment:
should we move this block of css to `Footer = styled.div`?

Reply:
Tried. Seems not working for some reason.
Apart from absolute bottom attribute, other relative attributes are ignored
<img width="1148" alt="Screen Shot 2019-04-21 at 12 31 42 am" src="https://user-images.githubusercontent.com/20808865/56458655-dc2e2280-63cc-11e9-8db6-483134d98f59.png">

Reply:
so it is working with the current set up?

Reply:
yes, it's working only if I put the inner div style in the react component

Reply:
yea. i tried a couple ways. no luck yet. we can revisit this later

Reply:
It's fixed now. It was because different syntax of css in react and styled component. easy to be overlooked when copy/paste/modify. I've merged the minor changes to master directly